src/library.rs

```diff
@@ -554,6 +555,50 @@ impl<Config: AppConfigTrait> Library<Config> {
         Ok(songs)
     }
 
+    /// Build a playlist of `playlist_length` items from a set of already analyzed
+    /// song in the library at `song_paths`, using distance metric `distance`,
+    /// the sorting function `sort_by` and deduplicating if `dedup` is set to
+    /// `true`.
+    ///
+    /// You can use ready to use distance metrics such as
+    /// [extended_isolation_forest], and ready to use sorting functions like
+    /// [closest_to_selected_songs_by_key].
+    pub fn playlist_from_many_custom<F, G, T: Serialize + DeserializeOwned + std::fmt::Debug>(
```

Comment:
It looks like you may have reviewed an earlier commit than the last one, unfortunately. In the last commit there is just `playlist_from` and `playlist_from_custom`, and I edited the existing tests to use those functions (passing a slice containing a single song instead of passing a single song).

Reply:
Ah, you're right, I did review commit by commit, disregard this comment. My follow-up question then is, is the `DistanceMetricBuilder` for `ForestOptions` used anywhere? (https://github.com/Polochon-street/bliss-rs/pull/72/files#diff-294648967369087cf3619793bd50043a2b39b5f481523d7c1636cc3e8d572a5bR84-R98)
Because I think it would be nice to have a small test that uses it - just to validate that it outputs proper stuff :)